src/Microsoft.TemplateEngine.Core.Contracts/PublicAPI.Shipped.txt

```diff
@@ -16,7 +16,6 @@ Microsoft.TemplateEngine.Core.Contracts.IOperation
 Microsoft.TemplateEngine.Core.Contracts.IOperation.IsInitialStateOn.get -> bool
```

Comment:
@baronfel: generic question - do we want to announce breaking API changes somewhere? 
We have certain API documented in .NET API browser and some are not (as this one).

Reply:
We lack comprehensive documentation overall for the API, right? So this is primarily something that will surface in the SDK and in the VS integration? I think it is ok to do this - NuGet.org doens't show any other consumers of this assembly aside from our own libraries.